packages/components-js/package.json

```diff
@@ -31,9 +31,12 @@
conLinks({ icons: [\"360\",\"accessibility\",\"active-cabin-ventilation\",\"add\",\"adjust\",\"arrow-double-down\",\"arrow-double-left\",\"arrow-double-right\",\"arrow-double-up\",\"arrow-down\",\"arrow-first\",\"arrow-head-down\",\"arrow-head-left\",\"arrow-head-right\",\"arrow-head-up\",\"arrow-last\",\"arrow-left\",\"arrow-right\",\"arrow-up\",\"augmented-reality\",\"battery-empty\",\"battery-full\",\"bell\",\"bookmark\",\"broadcast\",\"calculator\",\"calendar\",\"camera\",\"car\",\"car-battery\",\"card\",\"charging-active\",\"charging-state\",\"charging-station\",\"chart\",\"chat\",\"check\",\"city\",\"climate\",\"climate-control\",\"clock\",\"close\",\"closed-caption\",\"co2-emission\",\"compare\",\"configurate\",\"copy\",\"country-road\",\"cubic-capacity\",\"delete\",\"disable\",\"document\",\"download\",\"duration\",\"edit\",\"email\",\"error-filled\",\"exclamation\",\"external\",\"filter\",\"fingerprint\",\"flash\",\"fuel-station\",\"garage\",\"gift\",\"globe\",\"grid\",\"heart\",\"heart-filled\",\"highway\",\"home\",\"horn\",\"image\",\"increase\",\"information\",\"information-filled\",\"key\",\"leaf\",\"leather\",\"light\",\"list\",\"locate\",\"lock\",\"lock-open\",\"logo-apple-podcast\",\"logo-baidu\",\"logo-delicious\",\"logo-digg\",\"logo-facebook\",\"logo-foursquare\",\"logo-gmail\",\"logo-google\",\"logo-hatena\",\"logo-instagram\",\"logo-kaixin\",\"logo-kakaotalk\",\"logo-linkedin\",\"logo-naver\",\"logo-pinterest\",\"logo-qq\",\"logo-qq-share\",\"logo-reddit\",\"logo-skyrock\",\"logo-sohu\",\"logo-spotify\",\"logo-tecent\",\"logo-telegram\",\"logo-tiktok\",\"logo-tumblr\",\"logo-twitter\",\"logo-viber\",\"logo-vk\",\"logo-wechat\",\"logo-weibo\",\"logo-whatsapp\",\"logo-xing\",\"logo-yahoo\",\"logo-youku\",\"logo-youtube\",\"logout\",\"map\",\"menu-dots-horizontal\",\"menu-dots-vertical\",\"menu-lines\",\"minus\",\"mobile\",\"moon\",\"oil-can\",\"parking-brake\",\"parking-light\",\"pause\",\"phone\",\"pin\",\"pin-filled\",\"play\",\"plug\",\"plus\",\"preheating\",\"printer\",\"purchase\",\"push-pin\",\"push-pin-off\",\"qr\",\"question\",\"racing-flag\",\"refresh\",\"replay\",\"reset\",\"roof-closed\",\"roof-open\",\"route\",\"rss\",\"save\",\"screen\",\"search\",\"send\",\"share\",\"shopping-bag\",\"shopping-bag-filled\",\"shopping-cart\",\"shopping-cart-filled\",\"sidelights\",\"snowflake\",\"sort\",\"stack\",\"star\",\"steering-wheel\",\"stopwatch\",\"subtract\",\"success\",\"success-filled\",\"sun\",\"switch\",\"tablet\",\"tachometer\",\"tire\",\"truck\",\"upload\",\"user\",\"user-filled\",\"user-group\",\"user-manual\",\"video\",\"view\",\"view-off\",\"volume-off\",\"volume-up\",\"warning\",\"warning-filled\",\"weight\",\"wifi\",\"work\",\"wrench\",\"wrenches\",\"zoom-in\",\"zoom-out\"] }).replace(/https:\\/\\/cdn\\.ui\\.porsche\\.com\\/porsche-design-system/g, \"http://localhost:3001\"))') && regex=$placeholder'.*' && sed -E -e \"s@$regex@$partial@\" public/index.html > public/index.tmp.html && yarn replace:renameIndexHtml",
```

Comment:
Same here, in addition aren't there only like 5 icons that repeat themselves? All the others are probably pointless.